src/Http/Services/Contracts/FileServiceContract.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace EscolaLms\Files\Http\Services\Contracts;
+
+use Exception;
+
+interface FileServiceContract
+{
+    /**
+     * @param string $directory directory under which to check files for
+     * @param array $list List of Files
+     * @return boolean
+     */
+    function findAll(string $directory, array $list);
+
+    /**
+     * @param string $directory directory under which to file put in
+     * @param array $list List of Files
+     * @throws Exception
+     */
+    function putAll(string $directory, array $list);
```

Comment:
Let's be consistent if there is a public function in the other files, it should be so here.